translate/storage/rc.py

```diff
@@ -125,6 +125,7 @@ def __init__(self, inputfile=None, lang=None, sublang=None, **kwargs):
         if inputfile is not None:
             rcsrc = inputfile.read().decode(self.encoding)
             inputfile.close()
+            rcsrc = rcsrc.replace('\r', '')
```

Comment:
Wouldnt' it be better to adjust parser to treat '\r' same as '\n'? It already accepts '\r' in few places....

This way it needs double memory for replace and sole '\r' would be ignored, what looks like a bug to me.

Reply:
This is intentional (note the last testcase).
Visual studio (tested on 2015) would not compile an RC file if it used only '\r'

Reply:
But when the compiler does not parse it then translate-toolkit should not parse it as well, or am I missing something?

Reply:
Which is why I removed all \r characters, so that the file would be parsed roughly the same as with the visual studio compiler (all as one line).

Reply:
What I think compiler actually does is parse only '\r\n' as newline, can you check that only '\n' works? (I'm not on Windows...)

Reply:
I did, before writing the tests.
MSVC has been fine with just '\n' for years....